mixins/dynamicComponentHeight.ts

```diff
@@ -0,0 +1,18 @@
+import { isServer } from '@vue-storefront/core/helpers';
+
+export default {
+  data () {
+    return {
+      dynamicComponentHeight: 0
```

Comment:
can we change the name to `dynamicWindowHeight`? It's not really a component height